### ISSUE TITLE
Add some discovery features

### ISSUE THREAD

**Issue by Jomyy** [Iamlooker/Droid-ify#2]
a bit more like a store and less like a big list of apps

**Comment by Jomyy** [comment]
Also i can make some mockups or even fork the repo and make an example im just not at home atm but maybe next week

**Comment by jcperil** [comment]
If I understand correctly, and for clarity, you want the option of viewing apps in a horizontal grid (instead of a vertical list) with thumbnails for each app. And inside each app, you want a section recommending you similar apps. Please correct me if that is not the case.

**Comment by Jomyy** [comment]
Yes that could be one way, i mean look at the playstore designwise

**Comment by vertigo220** [comment]
One feature I can think of for this would be showing similar apps. That's often helpful in finding apps when I'm looking for something specific and sometimes even just to come across something unexpectedly.
